src/main/java/org/folio/config/RMAPIConfigurationClient.java

```diff
@@ -0,0 +1,97 @@
+package org.folio.config;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import org.folio.http.ConfigurationClientProvider;
+import org.folio.rest.client.ConfigurationsClient;
+import org.folio.rest.tools.client.Response;
+import org.folio.rest.tools.utils.TenantTool;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Retrieves the RM API connection details from mod-configuration.
+ */
+public class RMAPIConfigurationClient {
+
+  private static final String EBSCO_URL_CODE = "kb.ebsco.url";
+  private static final String EBSCO_API_KEY_CODE = "kb.ebsco.apiKey";
+  private static final String EBSCO_CUSTOMER_ID_CODE = "kb.ebsco.customerId";
+  private static final int QUERY_LIMIT = 100;
+
+  private ConfigurationClientProvider configurationClientProvider;
+
+  /**
+   * @param configurationClientProvider object used to get http client for sending request to okapi
+   */
+  public RMAPIConfigurationClient(ConfigurationClientProvider configurationClientProvider) {
+    this.configurationClientProvider = configurationClientProvider;
+  }
+
+  /**
+   * Returns the RM API configuration for the tenant.
+   *
+   * @param apiToken token used to call okapi
+   * @return The RMI API configuration for the tenant.
+   */
+  public CompletableFuture<RMAPIConfiguration> retrieveConfiguration(String apiToken, String tenant, String okapiURL) {
```

Comment:
Could you please update java doc with info about other parameters (tenant, okapiURL)